src/brain/issueLabelHandler/route.ts

```diff
@@ -131,36 +162,6 @@ async function routeIssue(octokit, productAreaLabelName) {
   }
 }
 
-async function getReadableTimeStamp(timeToTriageBy, productAreaLabelName) {
-  const dueByMoment = moment(timeToTriageBy).utc();
-  const officesForProductArea = await getSortedOffices(productAreaLabelName);
-  let lastOfficeInBusinessHours;
-  (officesForProductArea.length > 0 ? officesForProductArea : ['sfo']).forEach(
-    (office) => {
-      if (isTimeInBusinessHours(dueByMoment, office)) {
-        lastOfficeInBusinessHours = office;
-      }
-    }
-  );
-  if (lastOfficeInBusinessHours == null) {
-    lastOfficeInBusinessHours = 'sfo';
-    Sentry.captureMessage(
-      `Unable to find an office in business hours for ${productAreaLabelName} for time ${timeToTriageBy}`
-    );
-  }
-  const officeDateFormat =
-    lastOfficeInBusinessHours &&
-    OFFICES_24_HOUR.includes(lastOfficeInBusinessHours)
-      ? 'dddd, MMMM Do [at] HH:mm'
-      : 'dddd, MMMM Do [at] h:mm a';
-  return {
-    readableDueByDate: dueByMoment
-      .tz(OFFICE_TIME_ZONES[lastOfficeInBusinessHours])
-      .format(officeDateFormat),
-    lastOfficeInBusinessHours,
-  };
-}
-
```

Comment:
Are there material changes to this function? It looks like it's just moving to a different place in the file.

Reply:
see https://github.com/getsentry/eng-pipes/pull/488#discussion_r1237446925 😄

Reply:
😁